indexd/app.py

```diff
@@ -4,6 +4,9 @@
 from .alias.blueprint import blueprint as indexd_alias_blueprint
 from .dos.blueprint import blueprint as indexd_dos_blueprint
 from .blueprint import blueprint as cross_blueprint
+
+from indexd.index.urls.v0 import urls as index_urls_blueprint
```

Comment:
syntactical consistency is important in general, can we make the import statement and file structure in the same fashion as other blueprints? same with other minor things like `with self.driver.session as session` instead of `with self.driver.session as sxn`

Reply:
I would say both—I think that's what @philloooo meant as well

Reply:
Cools, the urls and metadata are index queries, so I pictured them as sub packages under the index package, Will update accordingly